larray/core/axis.py

```diff
@@ -1194,6 +1195,34 @@ def align(self, other, join='outer'):
                 other = Axis(other)
             return other
 
+    def to_hdf(self, filepath, key, **kwargs):
```

Comment:
key should be optional. The question is the default value. Defaulting to self.name would be easiest, I guess.